wallet-gateway/remote/src/index.ts

```diff
@@ -22,6 +22,7 @@ const program = new Command()
     .version(pkg.version)
     .description('Run a remotely hosted Wallet Gateway')
     .option('-c, --config <path>', 'set config path', './config.json')
+    .option('-p, --port [port]', 'set port', '3030')
```

Comment:
chose the 3030 default somewhat arbitrarily... but `8080` is already taken by the Example dApp server when running, and `3000` by something else if running localnet